### PR TITLE
prefilling out email to login

### PR DESCRIPTION
**Pull request by rjsurati**

### CODE REVIEW

File: frontend/src/components/session/session_form.jsx [hunk 1]
```diff
@@ -5,13 +5,17 @@ class SessionForm extends React.Component {
   constructor(props) {
     super(props);
 
+    const search = props.location.search; // could be '?foo=bar'
+    const params = new URLSearchParams(search);
+    const foo = params.get('email'); // bar
```
Comment (nucat176) [inline frontend/src/components/session/session_form.jsx]: can we use a more accurate naming convention here? foo and bar don't really signify anything to someone reading the code

File: frontend/src/components/session/session_form.jsx [hunk 2]
```diff
@@ -27,7 +31,8 @@ class SessionForm extends React.Component {
   }
 
   componentDidMount() {
-    document.getElementById("text").value = "";
+    document.getElementById("text").value = this.state.email;
+    console.log("abc")
```
Comment (nucat176) [inline frontend/src/components/session/session_form.jsx]: no need to keep the console.log

File: frontend/src/components/session/session_form.jsx [hunk 3]
```diff
@@ -81,6 +86,7 @@ class SessionForm extends React.Component {
                       autoCapitalize="none"
                       className="text-field"
                       placeholder="Email address"
+		      defaultValue="foobar"
```
Comment (nucat176) [inline frontend/src/components/session/session_form.jsx]: spacing?

File: server/controllers/awsEmail.js [hunk 4]
```diff
@@ -32,7 +32,7 @@ module.exports = {
                 to: to,
                 subject: `Your friend ${options.fromAddress} gave you jobCents!`,
                 // text: '',
-                html: htmlTemplate.inviteHtml(options.challengeTitle, options.description, options.fromAddress, formatDollars(options.rewardAmount), options.participationUrl, options.company)
+                html: htmlTemplate.inviteHtml(to, options.challengeTitle, options.description, options.fromAddress, formatDollars(options.rewardAmount), options.participationUrl, options.company)
```
Comment (nucat176) [inline server/controllers/awsEmail.js]: are there any other placing calling the inviteHtml function? I want to make sure the addition of the new parameter at the front of the function call doesn't cause other uses to fail elsewhere in the code

Comment (rjsurati) [reply server/controllers/awsEmail.js]: inviteHtml appears to only be called in one place

**Pull request merged**